virtcontainers/utils/utils_linux.go

```diff
@@ -71,6 +72,12 @@ func FindContextID() (*os.File, uint64, error) {
 		return nil, 0, err
 	}
 
+	if runtime.GOARCH == "ppc64le" {
+		ioctlVhostVsockSetGuestCid = 0x8008AF60
```

Comment:
I don't understand why this value is different, just for ppc64le...? Can you point to the kernel source which shows why this is required as looking at `linux/vhost.h` from the comment above suggests it's a generic value.

Reply:
@jodh-intel : More info here: https://github.com/kata-containers/runtime/issues/1943#issuecomment-521159458

Reply:
> I don't understand why this value is different, just for ppc64le...? Can you point to the kernel source which shows why this is required as looking at `linux/vhost.h` from the comment above suggests it's a generic value.

Please see the kernel codes "#define _IOC_WRITE	4U"  under powerpc arch.